packages/components/src/color-palette/test/index.js

```diff
@@ -92,7 +92,7 @@ describe( 'ColorPalette', () => {
 			const isOpen = true;
 			const onToggle = jest.fn();
 
-			const renderedToggleButton = shallow(
+			const renderedToggleButton = mount(
```

Comment:
Unit tests were failing, I believe because the switch from a vanilla `button` to a `Flex as={button}` meant that Enzyme's shallow render couldn't find that `button` anymore. Changing to `mount` solves the problems (with the consequence of generating a larger snapshot).

Not sure if this is the cleanest solution, but I figured it'd be good enough — especially considering that we should eventually rewrite these tests to use `@testing-library` instead.